languages/csharp/exercises/concept/user-defined-exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+
+public class CalculationException : Exception
+{
+    public CalculationException(int operand1, int operand2, string message, Exception inner) : base(message, inner)
+    // TODO: complete the definition of the constructor
```

Comment:
```suggestion
```

Reply:
I have put this back in (afte taking it out, after review).  It's important to show that the base class's constructor should be called.